### DISCUSSION TITLE
Welcome to Beach-Bot Discussions!

### DISCUSSION THREAD

**Discussion by bsod2528** [BSOD2528/Beach-Bot#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zeus432** [comment]
Hi I'm Zeus, I contribute to this repo at times :peaceout:
